ddtrace/appsec/_api_security/api_manager.py

```diff
@@ -75,12 +72,18 @@ def __init__(self) -> None:
         log.debug("%s initialized", self.__class__.__name__)
         self._hashtable: collections.OrderedDict[int, float] = collections.OrderedDict()
 
+        from ddtrace.appsec import _processor as appsec_processor
+        import ddtrace.appsec._asm_request_context as _asm_request_context
```

Comment:
Both imports trigger the ctypes init?

Reply:
At first yes, but after the changes not anymore. But it's still allow to reduce the direct dependencies between the modules.